### PR TITLE
chore: [M3-6803] - Use latest `@sentry/react` package and enable performance monitoring

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝
- Updates our Sentry package and makes sentry more useful

## Major Changes 🔄
- Uses `@sentry/react` rather than `@sentry/browser` because it is better for our use case 📦
- Enables Sentry's performance monitoring 🔎
  - This will duplicate some things New Relic does but it will give us some very nice information that we can all have access to seeing
  - This will give us better observability and debugability

## Preview 📷
Screenshot 2023-06-28 at 6 15 33 PM

## How to test 🧪
- Set `REACT_APP_SENTRY_URL` in your `packages/manager/.env` (ask me if you need the value)
- Throw an error in a component
- Verify you see the error in Sentry

### CODE REVIEW

File: packages/manager/src/App.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ import { tokenEventHandler } from './queries/tokens';
 import { volumeEventsHandler } from './queries/volumes';
 import { ApplicationState } from './store';
 import { getNextThemeValue } from './utilities/theme';
-import { ErrorBoundary } from './components/ErrorBoundary';
+import { ErrorBoundary } from '@sentry/react';
```
Comment (bnussman-akamai) [inline packages/manager/src/App.tsx:37]: Switching out for Sentry's `ErrorBoundry` in hopes we catch the most errors possible. It works the same as ours and will render the fallback as expected.

File: packages/manager/src/initSentry.ts [hunk 2]
```diff
@@ -12,6 +17,7 @@ export const initSentry = () => {
       environment: import.meta.env.PROD ? 'production' : 'development',
       beforeSend,
       autoSessionTracking: false,
+      integrations: [new BrowserTracing()],
```
Comment (bnussman-akamai) [inline packages/manager/src/initSentry.ts:20]: Enables Sentry's performance feature 🎉

File: packages/manager/src/App.tsx (hunk 1 above)
Comment (jaalah-akamai) [reply packages/manager/src/App.tsx:37]: Do we need to do this in the other 2 places we use our `ErrorBoundary`?

**Review by jaalah-akamai — APPROVED** [review]
🚀 📈

**Review by dwiley-akamai — APPROVED** [review]

File: packages/manager/src/App.tsx (hunk 1 above)
Comment (bnussman-akamai) [reply packages/manager/src/App.tsx:37]: Good point. I'll just switch things over to the Sentry error boundy...

**Pull request merged**